docs/resources/client-start-help

```diff
@@ -25,7 +26,7 @@ Flags:
       --tbtc.preParamsGenerationTimeout duration   tECDSA pre-parameters generation timeout. (default 2m0s)
       --tbtc.preParamsGenerationDelay duration     tECDSA pre-parameters generation delay. (default 10s)
       --tbtc.preParamsGenerationConcurrency int    tECDSA pre-parameters generation concurrency. (default 1)
-      --tbtc.keyGenerationConcurrency int          tECDSA key generation concurrency. (default number of cores)
+      --tbtc.keyGenerationConcurrency int          tECDSA key generation concurrency. (default 16)
```

Comment:
Let's revert this change, the default should be "number of cores".

There is a problem with a code generating this file as it is generating it based on the current environment.

Reply:
Done in 5419dc02e353b706fa483719b947842f58b680f3.